File: smallpict-php/src/Models/FitMode.php
<?php

declare(strict_types=1);

namespace SmallPict\Models;

final class FitMode
{
    public const COVER = 'cover';
    public const CONTAIN = 'contain';
    public const INSIDE = 'inside';
    public const OUTSIDE = 'outside';

    private function __construct()
    {
    }
}

File: smallpict-java/src/main/java/com/smallpict/models/FitMode.java
package com.smallpict.models;

import com.fasterxml.jackson.annotation.JsonValue;

public enum FitMode {
    COVER("cover"),
    CONTAIN("contain"),
    INSIDE("inside"),
    OUTSIDE("outside");

    private final String value;

    FitMode(String value) {
        this.value = value;
    }

    @JsonValue
    public String getValue() {
        return value;
    }

    @Override
    public String toString() {
        return value;
    }
}
